internal/command/command_plugin.go

```diff
@@ -67,7 +68,7 @@ func (cp *CommandPlugin) Info() *bus.Info {
 	}
 }
 
-// nolint: revive
+// nolint: revive, cyclop
```

Comment:
cyclop? Can you place case bus.DataPlaneResponseTopic: into it's own function that's testable?

Reply:
fixed